### ISSUE TITLE
Allow creation of Tracks in the app

### ISSUE THREAD

**Issue by srv-twry** [fossasia/open-event-orga-app#871]
### Feature Request

**Feature Description**

The app should support creation of Tracks. I propose to implement it in a similar way to the Creation of an Event.

**Would you like to work on the issue?**

Yes

**Parent issue**
#870 

To be done after #872

**Update**

~~~Busy till 13th April 2018(Friday) hence can be taken by someone else before that, Otherwise i will work on it on the weekend.~~~
Working on it!

**Comment by srv-twry** [comment]
@iamareebjamal The form for the Creation of Track needs to have entries for ```color``` and ```font-color``` which would need a color picker.
Which one of the two would you recommend using ? 
1. [First](https://github.com/QuadFlask/colorpicker)
2. [Second](https://github.com/eltos/SimpleDialogFragments)
or any other.

**Comment by iamareebjamal** [comment]
We want this to happen. Each time the create ticket fragment is opened, a new color is generated randomly but it should be bright or dark enough. With the corresponding contrasting color present as text color.

The ability to change or randomly generate other color should be there. If the color is clicked, we will show a dialog to pick color from https://github.com/Pes8/android-material-color-picker-dialog

But this can be done in separate PR. Right now, just handle the manual entry for these

**Comment by sridharjajoo** [comment]
@iamareebjamal What I could infer from above is that when the user enters the CreateTracks fragment then there should be a hex code already present in the color's edit text and it shoudn't be blank as it is currently?

**Comment by iamareebjamal** [comment]
Yes, both for track color and track text color
